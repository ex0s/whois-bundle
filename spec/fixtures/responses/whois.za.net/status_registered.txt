
Domain Name            : karnaugh.za.net

Registered for         : Colin Alston
                         11 Swales Crescent
                         Pinetown
                         3610
                         South Africa

Administrative Contact : Colin Alston <diskbox@yifan.net>
                         DAC
                         11 Swales Crecent
                         Pinetown
                         KZN
                         3610
                         South Africa
                         7037634

Technical Contact      : Colin Alston <diskbox@yifan.net>
                         DAC
                         11 Swales Crecent
                         Pinetown
                         KZN
                         3610
                         South Africa
                         7037634

Record Created         : Fri Mar 29 22:03:53 GMT 2002
Record Last Updated    : Fri Mar 29 22:03:53 GMT 2002
Database Last Updated  : Sun Nov 22 16:01:16 GMT 2009

Domain Name Servers listed in order:

  ns3.zoneedit.com                                    
  ns5.zoneedit.com                                    
                                                      

The ZA NiC whois service contains ONLY information relating to the registration
or availability of Internet domain names under the ZA.NET and ZA.ORG namespace.
Online registration of ZA NiC domains can be made using the automated tool
available at http://www.za.net/

