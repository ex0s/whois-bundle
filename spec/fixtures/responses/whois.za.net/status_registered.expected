#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2002-03-29 22:03:53 UTC}

#updated_on
  %s %CLASS{time}
  %s %TIME{2002-03-29 22:03:53 UTC}

#expires_on
  %s %ERROR{AttributeNotSupported}


#nameservers
  %s %CLASS{array}
  %s %SIZE{2}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns3.zoneedit.com"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns5.zoneedit.com"
